src/components/style/interactive/buttonGroup.css

```diff
@@ -9,35 +9,35 @@
 
 .btngroup {
   display: inline-flex;
-  z-index: 0;
+  z-index: var(--zindex-main);
   position: relative;
 }
 
 .btngroup .groupbtn {
   margin-left: -1px;
   position: relative;
   border-radius: 0;
-  z-index: 2;
+  z-index: calc(var(--zindex-main) + 1);
```

Comment:
In cbi-site we may see something like: `z-index: calc(var(--zindex-main) + 99999);` Lol

Reply:
🙂Technically you're not allowed to add more than 9. We're planning to go fix all the z-index values up in cbi-site, and then hopefully we'll catch anyone trying to do that in code review (or add some sort of lint rule).